Doc 4374 - Added 4.9.5 Documentation (#777)
corda/corda-docs-portal: content/en/platform/corda/4.9/community/release-notes.md

# Corda Community Edition 4.9 release notes

## Corda Community Edition 4.9.5 release notes

Corda Community 4.9.5 is a patch release of Corda Community which includes resolved issues.

### Fixed issues

> * Previously, a memory leak in the transaction cache occurred due to weight of in-flight entries being undervalued. Improvements to prevent in-flight entry weights from being undervalued and estimated more correctly means a large decrease in the total size of cached entities. 

Reviewer: ```suggestion
* Previously, a memory leak in the transaction cache occurred due to the weight of in-flight entries being undervalued. Improvements have been made to prevent in-flight entry weights from being undervalued and because they are now estimated more correctly, this results in a large decrease in the total size of cached entities. 
```

Author: Done.

> * A rare condition was found, when database transactions are rolled back under heavy load, that causes flow state machine threads to stop processing flows, leading to eventual node lock up in certain circumstances. This fix prevents this from happening.

Reviewer: ```suggestion
* A rare condition was fixed relating to roll back of database transactions under heavy load, which caused flow state machine threads to stop processing flows, leading to eventual node lock up in certain circumstances.
```

Author: Done.

### Database Schema Changes

* The `node_named_identities` table has been re-introduced. It was removed in Corda Enterprise Edition 4.7 following updates to certificate rotation functionality.
  

>   The reintroduction of this table ensures the behavior of `rpcOps.wellKnownPartyFromX500Name` is identical for both revoked and non-revoked identities.

Reviewer: ```suggestion
  The reintroduction of this table ensures that the behavior of `rpcOps.wellKnownPartyFromX500Name` is identical for both revoked and non-revoked identities.
```

Author: Done.
